RadarSDK/RadarVerificationManager.m

```diff
@@ -31,8 +31,12 @@
 
 @interface RadarVerificationManager ()
 
-@property (strong, nonatomic) NSTimer *timer;
+@property (assign, nonatomic) BOOL started;
+@property (strong, nonatomic) NSTimer *intervalTimer;
 @property (nonatomic, retain) nw_path_monitor_t monitor;
+@property (strong, nonatomic) RadarVerifiedLocationToken *lastToken;
+@property (assign, nonatomic) NSTimeInterval lastTokenSystemUptime;
+@property (assign, nonatomic) BOOL lastTokenBeacons;
```

Comment:
Nit: I think `latest` for these 3 might be slightly better verbiage here. "last" to me first parses as the final element in a list and sorta sounds like it's static / won't change.

Reply:
Could go either way here. fwiw we use `lastLocation`, `lastMovedAt`, etc. in `RadarState`, was following that naming convention